README.md

```diff
@@ -47,30 +47,37 @@ via lektor (see [Deployment](#deployment), below).
 
 ### Pre-Requisites
 
-Make sure you have [pipenv][pipenvdocs], [Node.js][nodejswebsite] and [npm][npmdocs] installed. To install these, execute the following commands:
+Make sure you have:
+
+* [pipenv][pipenvdocs]
+
+* [Node.js][nodejswebsite] **v12+** and [npm][npmdocs] installed. 
+
+To install these, execute the following commands:
 
 > For  macOS:
+
 1. `brew install pipenv`
 2. `brew install node`
 
 > For GNU/Linux:
+
 1. `curl https://raw.githubusercontent.com/kennethreitz/pipenv/master/get-pipenv.py | python`
 2. `sudo apt update`
-3. `sudo apt install nodejs`
-4. `sudo apt install npm `
+3. `curl -sL https://deb.nodesource.com/setup_lts.x | sudo -E bash -`
+4. `sudo apt-get install -y nodejs`
+5. `sudo apt install npm `
```

Comment:
Not everyone is on a Debian-derivative distro. Can we just point to the NodeSource page which has instructions for all supported Linux distros?